backend/src/types/models/case-files/wildlife/drug-used-input.ts

```diff
@@ -11,3 +11,15 @@ export interface DrugUsedInput {
   amountDiscarded: string;
   discardMethod: string;
 }
+
+export interface DrugUsedInputV2 {
+  id?: number;
+
+  vial: string;
+  drug: string;
+  amountUsed: string;
+  injectionMethod: string;
+
+  remainingUse: string;
+  additionalComments: string;
+}
```

Comment:
Isn't this creating dead code as no one else is using DrugUsedInput?

Reply:
Cleaned up the types with b5d83d8.